app/src/WalletProvider.tsx

```diff
@@ -116,6 +119,9 @@ function WalletProvider(props: React.PropsWithChildren<Record<string, never>>) {
               setNfts(data.assets);
             }
           })
+          .catch(e => {
+            setOpenseaError(e);
```

Comment:
To type-match this to the string let's set this as: `setError('Unable to search wallet using OpenSea for NFTs.');`